extensions/amp-access/0.1/amp-access-vendor.js

```diff
@@ -41,6 +41,12 @@ export class AccessVendorAdapter {
     this.vendorName_ = user().assert(configJson['vendor'],
         '"vendor" name must be specified');
 
+    /** @const @private {JSONType} */
+    this.vendorConfig_ = configJson[this.vendorName_];
+
+    /** @const @private {boolean} */
+    this.isPingbackEnabled_ = !configJson['noPingback'];
```

Comment:
I ended up [changing that on the service](https://github.com/ampproject/amphtml/pull/5920/commits/356edb33e2b77fed233deb67701b23542735931c), but I thought it should still make sense to return the configuration value here.